src/equations/compressible_euler_2d.jl

```diff
@@ -1252,6 +1252,99 @@ function flux_hllc(u_ll, u_rr, orientation::Integer,
     return SVector(f1, f2, f3, f4)
 end
 
+function flux_hllc(u_ll, u_rr, normal_direction::AbstractVector,
```

Comment:
Could you please adapt the docstring of the method above to indicate that it works for both the Cartesian and curved version?